docs/src/code-examples/generate-template-preview/java.mdx

```diff
@@ -0,0 +1,40 @@
+---
+lang: java
+---
+
+```java
+TemplatePreview templatePreview = client.generateTemplatePreview(
+    templateId,
+    personalisation
+);
+```
+
+The parameters in the personalisation argument must match the placeholder fields in the actual template.
+The Java client ignores any extra fields in the method.
+
+<Panel label={<H3 appearAs={H5}>Arguments</H3>}>
+  <Parameters>
+    <Parameter required name="templateId" kind="String">
+      Sign in to Notify.gov.au and go to the Templates page to find the template
```

Comment:
@andrewoh531 as long as most of the docs already say `Notify` instead of `Notify.gov.au`, then maybe prefer `Notify`?